packages/cache/__tests__/cacheUtils.test.ts

```diff
@@ -32,3 +32,8 @@ test('assertDefined throws if undefined', () => {
 test('assertDefined returns value', () => {
   expect(cacheUtils.assertDefined('test', 5)).toBe(5)
 })
+
+test('resolvePaths works on current directory', async () => {
+  const paths = await cacheUtils.resolvePaths(['.'])
```

Comment:
it would be better to set the workspace path as the param here and check that the result is not empty.